gui/index.go

```diff
@@ -142,3 +142,35 @@ func (ui *GUI) Homepage(w http.ResponseWriter, r *http.Request) {
 
 	ui.renderTemplate(w, "account", data)
 }
+
+// DoesAccountExist is the handler for "GET /account". If the provided address
+// has an account on the server a "200 OK" response is returned, otherwise a
+// "400 Bad Request" is returned.
+func (ui *GUI) DoesAccountExist(w http.ResponseWriter, r *http.Request) {
```

Comment:
Rename this to `IsPoolAccount` instead, it's a better name.

Reply:
Done